monai/transforms/croppad/array.py

```diff
@@ -176,10 +189,10 @@ def update_meta(self, tensor: MetaTensor, to_pad: List[Tuple[int, int]]):
     def inverse(self, data: MetaTensor) -> MetaTensor:
         transform = self.pop_transform(data)
         padded = transform[TraceKeys.EXTRA_INFO]["padded"]
-        if padded[0][0] != 0 or padded[0][1] != 0:
-            raise NotImplementedError(
-                "Inverse uses SpatialCrop, which hasn't yet been extended to crop channels. Trivial change."
-            )
+        if padded[0][0] > 0 or padded[0][1] > 0:  # slicing the channel dimension
+            s = [padded][0][0]
+            e = min(max([padded][0][1], s + 1), len(data))
```

Comment:
I don't quite understand the change here, what do you mean `slicing the channel dimension`?
And what's the difference between `[padded][0]` and `padded`?

Thanks.

Reply:
padded[0] is the potential padding for the first dimension (channel), they are currently zeros in our spatial cases. But if the channel-wise padding are not zeros, this logic will inverse it by slicing the channels.

Reply:
Hi @wyli , I mean why adding `[ ]` to the `padded` var then slice the first item `[padded][0]`?

Thanks.